### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/cirrus-ci-web/commit/6f3fa85851f54204618cc642991b82d192f33955#commitcomment-87783066

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/cirrus-ci-web/commit/3f59d97e9660f607367f259a8a51c3c383fdce80

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: src/components/builds/BuildDurationsChart.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ function BuildDurationsChart(props: Props) {
           y={y - sign * 2}
           height={height + sign * 2}
           fill={statusColorMapping[props.status]}
-          classbaName="recharts-bar-rectangle"
+          className="recharts-bar-rectangle"
```
Comment (jsoref) [inline src/components/builds/BuildDurationsChart.tsx:53]: scary

File: src/components/cirrus-terminal/api/terminal_pb.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ goog.exportSymbol('proto.TerminalDimensions', null, global);
 /**
  * Generated by JsPbCodeGenerator.
  * @param {Array=} opt_data Optional initial data array, typically from a
- * server response, or constructed directly in Javascript. The array is used
+ * server response, or constructed directly in JavaScript. The array is used
```
Comment (jsoref) [inline src/components/cirrus-terminal/api/terminal_pb.js:41]: Brand

**Review by fkorotkov — APPROVED** [review]

**Comment by fkorotkov** [comment]
Thank you, @jsoref! Do you have an option for `check-spelliing` to be distributed as a container that can output some machine readable output with the issues? We don't use GHA.

**Comment by jsoref** [comment]
It's possible to run it using nektos/act. Which basically runs it in docker....

I'm not sure if there's a consumer actively doing this, bit I know I've suggested it.

**Pull request merged**